atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -236,10 +238,32 @@ def _make_labels(self):
 
         return labels
 
+    def get_fs_task_id(self):
+        task_id = None
+        fs_result = self.workflow.prebuild_results.get(AddFilesystemPlugin.key)
+        if fs_result is not None:
+            try:
```

Comment:
Right, if there's a `fs_result`, meaning the `add_filesystem` plugin actually ran, and we can't grab the filesystem task ID something is wrong.